exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "TestSingleBookPrice",
+            "expected": 8.0
+         },
+         {
+            "description": "TestBasketContainsMyBooks",
```

Comment:
@petertseng  I agree.  I am going to clean this up a bit.  This initial version of the json file is essentialy just a dump of the tests.  I was not sure at the time what exactly should appear in the json file or what its purpose is.  As I have stated before this is my first time trying to add an exercise, so learning the ropes.